### ISSUE TITLE
`doas` doesn't work with sudoloop

### ISSUE THREAD

**Issue by NChechulin** [Morganamilo/paru#740]
### Affected Version
**paru -V**: `paru v1.10.0 - libalpm v13.0.1`

### Description
**Have you checked previous issues?** Yes

Paru seems to ask for password in a loop, even though I am sure the password is correct. As soon as I turn off the `SudoLoop` option, everything is fine

### Output

**paru.conf**:
```
[options]
PgpFetch
Devel
Provides
DevelSuffixes = -git -cvs -svn -bzr -darcs -always -hg
#AurOnly
#BottomUp
#RemoveMake
SudoLoop = true
#UseAsk
#SaveChanges
#CombinedUpgrade
#CleanAfter
#UpgradeMenu
#NewsOnUpgrade
BatchInstall
Limit = 50
SkipReview

#LocalRepo
#Chroot
#Sign
#SignDb

[bin]
FileManager = ranger
#MFlags = --skippgpcheck
Bat = bat
Sudo = doas
```
If I comment the whole `SudoLoop` line, everything works.
If I remove the `= true` from the line, `doas` throws some error: `doas: invalid option -- 'v'`
If I leave it as it is, `doas` asks for password twice and then prints the "Authentication failed" message:

```
doas (nikolay@laptop) password: <correct password>
doas (nikolay@laptop) password: <correct password>
doas: Authentication failed
```

I use `doas` daily, so I am sure that it is configured properly and works fine.

**Comment by NChechulin** [comment]
Update: works fine with `paru -R`, doesn't work with `paru` and `paru -S`

**Comment by Morganamilo** [comment]
Does your doas actually persist?

**Comment by NChechulin** [comment]
> Does your doas actually persist? 

No, I did not turn on the feature. My opinion is that it should not be an issue. At least I see it that way

**Comment by Morganamilo** [comment]
I'm not sure how you expect sudoloop so work without persistence.
